<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostOptions</class>
 <widget class="QWidget" name="PostOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="EntryOptions_">
     <property name="title">
      <string>Entry options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="Mood_">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QComboBox" name="ScreenComments_">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QComboBox" name="Adult_">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="moodLabel">
        <property name="text">
         <string>Mood:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="Access_"/>
      </item>
      <item row="10" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Userpic:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="10" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="UserPic_"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="QGroupBox" name="LikeButtonsBox_">
        <property name="title">
         <string>Like buttons</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="AllLike_">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="VkontakteLike_">
           <property name="text">
            <string>Vkontakte</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="FacebookLike_">
           <property name="text">
            <string>Facebook</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="LiveJournalReward_">
           <property name="text">
            <string>Reward this (LiveJournal)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="LiveJournalRepost_">
           <property name="text">
            <string>LiveJournal</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="GoogleLike_">
           <property name="text">
            <string>Google+</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="TumblrLike_">
           <property name="text">
            <string>Tumblr</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="TwitterLike_">
           <property name="text">
            <string>Twitter</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="SurfingbirdLike_">
           <property name="text">
            <string>Surfingbird</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="whereIsYouLabel">
        <property name="text">
         <string>Location:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="accessLabel">
        <property name="text">
         <string>Access:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="Place_"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="Comments_">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="commentsLabel">
        <property name="text">
         <string>Comments:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="hideCommentsLabel">
        <property name="text">
         <string>Screening:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="AutoDetect_">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="musicLabel">
        <property name="text">
         <string>Music:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QCheckBox" name="NotifyAboutComments_">
        <property name="text">
         <string>Notify about replies</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QCheckBox" name="ShowInFriendsPage_">
        <property name="text">
         <string>Show in friends page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="Music_"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="adultContentLabel">
        <property name="text">
         <string>Adult content:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="2" colspan="2">
       <widget class="QLabel" name="UserPicLabel_">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>VkontakteLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>GoogleLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>LiveJournalReward_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>TwitterLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>FacebookLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>TumblrLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>LiveJournalRepost_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllLike_</sender>
   <signal>toggled(bool)</signal>
   <receiver>SurfingbirdLike_</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
